src/compiler/moduleSpecifiers.ts

```diff
@@ -570,6 +571,12 @@ function getLocalModuleSpecifier(moduleFileName: string, info: Info, compilerOpt
     return isPathRelativeToParent(maybeNonRelative) || countPathComponents(relativePath) < countPathComponents(maybeNonRelative) ? relativePath : maybeNonRelative;
 }
 
+function packageJsonPathsAreEqual(a: string | undefined, b: string | undefined, ignoreCase?: boolean) {
+    if (a === b) return true;
+    if (a === undefined || b === undefined) return false;
+    return comparePaths(a, b, ignoreCase) === Comparison.EqualTo;

```

Comment:
dont you need to pass current directory for this to work correctly

Reply:
These paths are already absolute, it seems, so nothing broke...

Reply:
Checked the other use in this file and they don't pass in the working dir either (also for paths from this host method)